projects/js-packages/charts/src/index.ts

```diff
@@ -37,4 +37,10 @@ export type { LineStyles, GridStyles, EventHandlerParams } from '@visx/xychart';
 
 export type { RenderLineStartGlyphProps } from './components/line-chart/line-chart';
 export type { LegendProps, BaseLegendProps, ChartLegendOptions } from './components/legend';
-export type { ConversionFunnelChartProps, FunnelStep } from './components/conversion-funnel-chart';
+export type {
```

Comment:
Following the discussion p1756422948449869/1756420336.932989-slack-C82FZ5T4G. Let's not export the types for particular chart from global name space - exporting from their own package is enough. In this case, the exports from projects/js-packages/charts/src/components/conversion-funnel-chart/index.ts would do it.